metalearn/metafeatures/perceptron_metafeatures.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+
+from sklearn.linear_model import Perceptron
+
+from metalearn.metafeatures.common_operations import profile_distribution
+
+
+def get_fitted_perceptron(X, Y, seed, n_classes, frac='all'):
+    class_fraction = n_classes/len(X)
+
+    if frac == 'all':
```

Comment:
Should `frac` be a float?

Reply:
It can be a float, but in order to include the option to use sqrt(n) instances of the data, we would still need some kind of indicator to use it. Should I make it a float with the option to pass in 'sqrt' as well?

Reply:
Is it not possible to set `frac=sqrt(n)`?

Reply:
I'm not sure, unless it's possible to do some computation before calling the function, because we would need to know the value of `n` before we pass it in. Maybe with Evan's new updates it is now possible, I'll have to check.

Reply:
You are right, we don't support computed arguments. One alternative would be to have wrapper functions that call this function and hard-code the value of `frac`. I don't think that buys us anything. Let's just leave this the way it is.